crates/core_simd/src/lane_count.rs

```diff
@@ -28,6 +32,15 @@ macro_rules! supported_lane_count {
         $(
             impl SupportedLaneCount for LaneCount<$lanes> {
                 type BitMask = [u8; ($lanes + 7) / 8];
+                const EMPTY_BIT_MASK: Self::BitMask = [u8::MIN; ($lanes + 7) / 8];
+                const FULL_BIT_MASK: Self::BitMask = {
+                    const LEN: usize = ($lanes + 7) / 8;
+                    let mut array = [u8::MAX; LEN];
```

Comment:
```suggestion
                const EMPTY_BIT_MASK: Self::BitMask = [0; ($lanes + 7) / 8];
                const FULL_BIT_MASK: Self::BitMask = {
                    const LEN: usize = ($lanes + 7) / 8;
                    let mut array = [!0u8; LEN];
```

imo literals are clearer since we care about which bits are set, rather than whatever the maximum/minimum is.

Reply:
I'm fine with that and agree: I mostly just copied from the existing code which preferred using `u8::MIN` and `u8::MAX`.

Reply:
Also following up on this, I also changed the `u8::MAX` below to be `!0`, and also the `&=` to just `=` since it's a bit clearer.